### DISCUSSION TITLE
Helm Deployment with Different Volume Servers

### DISCUSSION THREAD

**Discussion by ProjectInitiative** [seaweedfs/seaweedfs#2347] [Q&A]
Taking a look at the [README](https://github.com/chrislusf/seaweedfs/tree/master/k8s/helm_charts2#current-instances-config-aio), is there a suggested way to configure different host paths for different nodes? 

i.e. Volume node1 dir: /data-UUID1. Volume node2 dir:/data-UUID2 etc etc.

Looking at [values.yaml](https://github.com/chrislusf/seaweedfs/blob/master/k8s/helm_charts2/values.yaml#L163) are all nodes expected to have the same data directories? 

Thanks for any insight

**Comment by chrislusf** [comment]
The volume directory is local to the volume node. I suppose the local directories would be mounted to different host directories.

**Reply by ProjectInitiative** [reply]
> The volume directory is local to the volume node. I suppose the local directories would be mounted to different host directories.

I guess what I was getting at: I am planning on setting up a local-path-provisioner that specifies the pool of directories on each node available. I am not sure how to specify different volume nodes (i.e. volume1, volume2, volume3) 

Reading the README: "on the volume stateful set added support for K8S PVC, currently example with the simple local-path-provisioner from Rancher (comes included with k3d / k3s) https://github.com/rancher/local-path-provisioner"

Looking at the templates and values.yaml, I am not sure how to scale out to all nodes that have a sw-volume=true. Would that be as simple as scaling the replica count to the number of nodes that contain said label? 

From my understanding the replica/replicaset parameter creates instances of the "same volume server" rather than multiple distinct servers.
